src/core/base-runner.js

```diff
@@ -17,6 +17,7 @@ function toRunnable(plug) {
     console.warn("Plugin lacks name:", plug);
   }
   return config => {
+    // eslint-disable-next-line no-async-promise-executor
```

Comment:
I honestly think we should disable this globally. I’m of the option that this is not that bad, and can be quite useful.

Reply:
On the other hand, I guess having it enabled does protect us from naive uses of it 🤔 argh, ok let’s keep it.